### ISSUE TITLE
Upgrade from v1.10 to v1.11 failed on Redis (The ZADD command contains a "null" argument.)

### ISSUE THREAD

**Issue by zipang** [NodeBB/NodeBB#7350]
Here is the upgrade scripts log : 

```
OK | 13 script(s) found, 57 skipped
  → [2018/8/28] Record first entry in username/email history... OK
  → [2018/9/24] Upgrade bans to hashes... OK
  → [2018/9/28] Give global local login privileges... OK
  → [2018/10/1] Optimize PostgreSQL sessions... OK
  → [2018/10/4] add filters to events... OK
  → [2018/10/10] Fix category post zsets... OK
  → [2018/10/11] Fix category topic zsets...
    [#############] (20849/21310) 97% node_redis: Deprecated: The ZADD command contains a "null" argument.
This is converted to a "null" string now and will return an error from v.3.0 on.
Please handle this in your code to make sure everything works as you intended it to.
Error occurred
Error occurred during upgrade: ReplyError: ERR value is not a valid float
    at parseError (/nodebb/node_modules/redis-parser/lib/parser.js:193:12)
    at parseType (/nodebb/node_modules/redis-parser/lib/parser.js:303:14)

/nodebb/src/cli/upgrade.js:69
			throw err;
			^
ReplyError: ERR value is not a valid float
    at parseError (/nodebb/node_modules/redis-parser/lib/parser.js:193:12)
    at parseType (/nodebb/node_modules/redis-parser/lib/parser.js:303:14)
```

And the upgrade script failed to continue its process..! 😱

**Comment by zipang** [comment]
Hi @barisusakli  !
Thxxx for your reactivity ! :)
When is the 1.12 release due ?
I would have been glad to have a 1.11.x update, because this is happening inside the upgrade scripts... not in nodebb's core..

**Comment by barisusakli** [comment]
I will fix this and backport it to 1.11.x so you don't have to wait for the release.

**Comment by zipang** [comment]
Thxx a lot @barisusakli That was quick ! 👍 👏 
Now, how should i procede ?
Should i restore the database to its previous state, and relaunch the `./nodebb upgrade` script ?
